### DISCUSSION TITLE
Get string with all `packageFileDir`s

### DISCUSSION THREAD

**Discussion by nfelt14** [renovatebot/renovate#31215] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

Is there a way to get a string with all of the `packageFileDir`s that contain files that will be changed/updated? Currently the `packageFileDir` seems to only show the last found folder with a package in it, even if there are multiple files in separate directories that will be updated.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
There is not
